jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -1242,8 +1242,8 @@ static void initSecurityManager(ClassLoader applicationClassLoader) {
 /*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 public static void setSecurityManager(final SecurityManager s) {
 /*[IF CRIU_SUPPORT]*/
-	if (openj9.internal.criu.InternalCRIUSupport.isCRIUSupportEnabled()) {
-		/*[MSG "K0B02", "Enabling a SecurityManager currently unsupported when -XX:+EnableCRIUSupport is specified"]*/
+	if (openj9.internal.criu.InternalCRIUSupport.isCRaCorCRIUSupportEnabled()) {
```

Comment:
not sure this is the case for CRac

Reply:
we will need to check

Reply:
[CRaC System. setSecurityManager()](https://github.com/openjdk/crac/blob/93000a3cbf357983f9f74b5a15b1efd33f272368/src/java.base/share/classes/java/lang/System.java#L407) doesn't throw a `UnsupportedOperationException`.

Was the initial intention to prevent sensitive data captured in a checkpoint image?
Maybe no need to do so if `-XX:-CRIUSecProvider` is specified?

Reply:
`-XX:-CRIUSecProvider` should be default for crac. The existing policy is that the user will take care of security themselves

Reply:
So for crac there is no change in behaviour from a security perspective

Reply:
Restored `isCRIUSupportEnabled()` check.